definitions/npm/ramda_v0.26.x/flow_v0.104.x-/test_ramda_v0.26.x_list.js

```diff
@@ -822,4 +818,26 @@ const str: string = "hello world";
     s: a,
     y: b
   }))([1, 2, 3])(["1", "2", "3"]);
+
+  describe('nth', () => {
+    it('should pass', () => {
+      const nthxs1: ?number = nth(2, [1,2,3]);
+      const nthxs2: ?number = nth(2)([1,2]);
+      const nthxs3: ?string = nth(2, ["curry"]);
+      const nthxs4: ?string = nth(2)(["curry"]);
+      const nthxs5: string = nth(2)("curry");
+      const nthxs6: string = nth(2, "curry");
+    });
+
+    it('should fail', () => {
```

Comment:
The same goes for the negative cases - Why should 834 fail?

Reply:
Have done the change, missed your question, 834 was expecting a maybe string back, but in put is array of numbers, type mismatch, thus it should fail.